controllers/configurationpolicy_controller.go

```diff
@@ -853,6 +877,11 @@ func (r *ConfigurationPolicyReconciler) handleObjectTemplates(plc policyv1.Confi
 				objectT.ObjectDefinition.Raw = resolvedTemplate.ResolvedJSON
 			}
 		}
+
+		endTime := time.Now().UTC()
+		durationSeconds := float64(endTime.Sub(startTime)) / float64(time.Second)
```

Comment:
Nit: I think `time.Since(startTime).Seconds()` would be more clear, and could be more performant, although I don't think there would be a big difference.